src/silvimetric/storage.py

```diff
@@ -4,11 +4,16 @@
 from math import floor
 import pathlib
 import pyproj
+from time import sleep
+
+import asyncio
+from redis import Redis
+from pottery import Redlock
 
 class Storage(object):
     """ Handles storage of shattered data in a TileDB Database. """
 
-    def __init__(self, tdb_dir: str, ctx:tiledb.Ctx=None):
+    def __init__(self, tdb_dir: str, ctx:tiledb.Ctx=None, redis_url:str=None):
```

Comment:
we should merge my Storage/Configuration refactor before you get too far with this